src/lib/wallet/index.ts

```diff
@@ -64,7 +66,7 @@ export default class Wallet extends Web3Wallet implements HederaNativeWallet {
     privateKey: string,
     _provider?: Provider,
   ): HederaWallet {
-    const network = chainId.split(':')[1]
+    const network = chainId.split(':')[1];
```

Comment:
added this ticket regarding prettier rules - https://github.com/orgs/hgraph-io/projects/4?pane=issue&itemId=44613969